src/IceRpc.Slice.Generators/Internal/ServiceMethod.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) ZeroC, Inc.
+
+namespace IceRpc.Slice.Generators.Internal;
+
+/// <summary>Represents an RPC operation annotated with the <c>IceRpc.Slice.SliceOperationAttribute</c> attribute.
+/// </summary>
+internal record struct ServiceMethod
```

Comment:
Given the usage, I think it should be a readonly record struct.

Reply:
The generator targets `netstandard2.0` we cannot use `init` properties, aren't these required to make it readonly?

Reply:
Given the usage, you don't need init property either. You can just use the primary constructor. You could also simplify your record struct:
```
internal readonly record struct ServiceMethod(...);
```

Reply:
That doesn't work with .netstandard2.0, it results in:

```
Predefined type 'System.Runtime.CompilerServices.IsExternalInit' is not defined or imported[CS0518](https://msdn.microsoft.com/query/roslyn.query?appId%3Droslyn%26k%3Dk%28CS0518%29)
```